src/tests/ftest/util/command_utils.py

```diff
@@ -527,14 +547,14 @@ def run(self):
         """
         try:
             # This assigns self.result
-            super().run()
+            super().run(raise_exception)
         except CommandFailure as error:
             raise CommandFailure(
                 "<{}> command failed: {}".format(
                     self.command, error)) from error
         return self.result
 
-    def _get_result(self, sub_command_list=None, **kwargs):
+    def _get_result(self, sub_command_list=None, raise_exception=None, **kwargs):
```

Comment:
This function is missing one more edit.  On https://github.com/daos-stack/daos/blob/53e17d9b6eadee99707ce2d9266d15ceb78c67fe/src/tests/ftest/util/command_utils.py#L597 the line should be modified to be:
```
        return self.run(raise_exception)
```